Xamarin.Forms.Platform.WP8/FrameRenderer.cs

```diff
@@ -53,5 +55,15 @@ void UpdateBorder()
 			else
 				Control.BorderBrush = new Color(0, 0, 0, 0).ToBrush();
 		}
+
+		void UpdateCornerRadius()
+		{
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
+				cornerRadius = 5f; // default corner radius
```

Comment:
I hate hardcoded magic numbers too, but unfortunately this is how it was previously, see previous code. And I thought I shouldn't add more changes to the code.